### ISSUE TITLE
Build failing, probably due to syntex changes

### ISSUE THREAD

**Issue by nicolas-cherel** [LeoTestard/rustlex#31]
I can't figure out what's happening, it might be a bug from syntex, but since rustlex_codegen is impacted, opening an issue here is a good start point for investigation. Anyway, I really need help on this one !

https://travis-ci.org/nicolas-cherel/rumblebars/builds/74277576

```
   Running `rustc /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustlex_codegen-0.3.0/src/lib.rs --crate-name rustlex_codegen --crate-type lib -g --cfg feature="syntex_syntax" --cfg feature="syntex" --cfg feature="with-syntex" -C metadata=2633a99e76d1b9c3 -C extra-filename=-2633a99e76d1b9c3 --out-dir /home/travis/build/nicolas-cherel/rumblebars/target/debug/deps --emit=dep-info,link -L dependency=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps -L dependency=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps --extern quasi=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps/libquasi-2c2ffe75f8ada658.rlib --extern syntex_syntax=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps/libsyntex_syntax-dc97785e9d4b7cb6.rlib --extern log=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps/liblog-54cf393d3c69686f.rlib --extern syntex=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps/libsyntex-7395044252fcbb31.rlib -Awarnings`

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustlex_codegen-0.3.0/src/parser.rs:159:25: 159:40 error: unresolved name `token::get_name`

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustlex_codegen-0.3.0/src/parser.rs:159                 let s = token::get_name(id);

                                                                                                                             ^~~~~~~~~~~~~~~

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustlex_codegen-0.3.0/src/parser.rs:203:35: 203:50 error: unresolved name `token::get_name`

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustlex_codegen-0.3.0/src/parser.rs:203             match regex::string(&*token::get_name(id)) {

                                                                                                                                       ^~~~~~~~~~~~~~~

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustlex_codegen-0.3.0/src/parser.rs:217:21: 217:36 error: unresolved name `token::get_name`

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustlex_codegen-0.3.0/src/parser.rs:217                     token::get_name(id.name))))

                                                                                                                         ^~~~~~~~~~~~~~~

note: in expansion of format_args!

<std macros>:2:26: 2:57 note: expansion site

<std macros>:1:1: 2:61 note: in expansion of format!

/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustlex_codegen-0.3.0/src/parser.rs:216:18: 217:46 note: expansion site

error: aborting due to 3 previous errors

Could not compile `rustlex_codegen`.

Caused by:

  Process didn't exit successfully: `rustc /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/rustlex_codegen-0.3.0/src/lib.rs --crate-name rustlex_codegen --crate-type lib -g --cfg feature="syntex_syntax" --cfg feature="syntex" --cfg feature="with-syntex" -C metadata=2633a99e76d1b9c3 -C extra-filename=-2633a99e76d1b9c3 --out-dir /home/travis/build/nicolas-cherel/rumblebars/target/debug/deps --emit=dep-info,link -L dependency=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps -L dependency=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps --extern quasi=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps/libquasi-2c2ffe75f8ada658.rlib --extern syntex_syntax=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps/libsyntex_syntax-dc97785e9d4b7cb6.rlib --extern log=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps/liblog-54cf393d3c69686f.rlib --extern syntex=/home/travis/build/nicolas-cherel/rumblebars/target/debug/deps/libsyntex-7395044252fcbb31.rlib -Awarnings` (exit code: 101)
```

**Comment by bluejekyll** [comment]
same problem.

**Comment by kstep** [comment]
There are 3 related PRs with the fixes: #29, #30, #32. They are almost identical. Now we should wait for the repo owner to come here and merge one of them in.

**Comment by LeoTestard** [comment]
Fixed by #32.
